packages/datagateway-download/src/downloadApiHooks.test.tsx

```diff
@@ -542,4 +739,762 @@ describe('Download Cart API react-query hooks test', () => {
       );
     });
   });
+
+  describe('useDownloads', () => {
+    it('should retrieve user downloads', async () => {
+      const mockFormattedDownloadItems: FormattedDownload[] = [
```

Comment:
This is identical to `mockFormattedDownloadItems` set above - maybe it's worth changing the earlier assignments into `let`s and setting them to their defaults in a `beforeEach` and then modify the data as you require in each test? We can definitely shave off some lines of repeated mock definitions ;)

Reply:
Yeah good point, when I moved the mock items out I must have left out some local variables.